package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "parcellab-utilities",
-  "version": "1.5.6",
+  "version": "1.5.8",
```

Comment:
6 to 8? what is with 7?

Reply:
It is because we already have the tag named as version 1.5.7 (https://github.com/parcelLab/parcellab-utilities/releases/tag/v1.5.7)